pkg/ui/src/Header/Header.svelte

```diff
@@ -0,0 +1,54 @@
+<script lang="ts">
+	import Logo from './Logo.svelte';
+	export let title = 'Stock';
+	const links = [
+		{
+			label: 'Stock',
+			href: '/inventory/stock'
+		},
+		{
+			label: 'Inbound',
+			href: '/inventory/inbound'
+		},
+		{
+			label: 'Outbound',
+			href: '/inventory/outbound'
+		}
+	];
+	export let currentUrl = 'stock';
+</script>
+
+<div class="bg-gray-900 p-10 pt-5 ">
+	<div class="border-b border-sky-900 pb-5 flex sm:items-center ">
+		<Logo />
+
+		<div class="pl-4">
+			<nav class="flex space-x-4">
+				{#each links as { label, href }}
+					<a
+						{href}
+						class={[
+							'px-3 py-2 text-sm rounded-md',
+							currentUrl === label.toLowerCase()
+								? 'bg-teal-500 text-gray-900'
+								: 'text-white hover:text-teal-500'
+						].join(' ')}
```

Comment:
I know I've suggested this for react, but could we use `"px-3 py-2 text-sm rounder-md {/* current url check here */ ? "bg-teal-500 text-gray-900" : "text-white hover:text-teal-500"}"`. That way we would still get nice formatting, but also class sorting (to some extent). For full reasoning, check #121